### PR TITLE
Remove Postgres option from 'Attach Database Account...'

### PR DESCRIPTION
**Pull request by wwlorey**
Related to https://github.com/microsoft/vscode-cosmosdb/issues/1498#issuecomment-634195234

### CODE REVIEW

File: src/tree/AttachedAccountsTreeItem.ts [hunk 1]
```diff
@@ -121,7 +121,7 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
     }
 
     public async attachNewAccount(): Promise<void> {
-        const defaultExperiencePick = await vscode.window.showQuickPick(getExperienceQuickPicks(), { placeHolder: "Select a Database Account API...", ignoreFocusOut: true });
```
Comment (EricJizbaMSFT) [inline src/tree/AttachedAccountsTreeItem.ts:124]: We don't support attach for postgres? Is there an issue for that?

File: src/AzureDBExperiences.ts [hunk 2]
```diff
@@ -95,4 +99,5 @@ const GremlinExperience: Experience = { api: API.Graph, longName: "Gremlin", des
 const PostgresExperience: Experience = { api: API.Postgres, longName: "PostgreSQL", shortName: "Postgres" };
 
 const experiencesArray: Experience[] = [CoreExperience, MongoExperience, TableExperience, GremlinExperience, PostgresExperience];
+const cosmosExperiencesArray: Experience[] = [CoreExperience, MongoExperience, TableExperience, GremlinExperience];
```
Comment (neelip) [inline src/AzureDBExperiences.ts:102]: nit:
```suggestion
const cosmosExperiencesArray: Experience[] = [CoreExperience, MongoExperience, TableExperience, GremlinExperience];
const experiencesArray: Experience[] = cosmosExperiencesArray;
experiencesArray.push(PostgresExperience);
```

File: src/tree/AttachedAccountsTreeItem.ts (hunk 1 above)
Comment (neelip) [reply src/tree/AttachedAccountsTreeItem.ts:124]: Not for this release, we haven't created support for it yet.

Comment (wwlorey) [reply src/tree/AttachedAccountsTreeItem.ts:124]: https://github.com/microsoft/vscode-cosmosdb/issues/1531

**Pull request merged**